dev/TreeView/TreeViewItem.xaml

```diff
@@ -109,6 +116,20 @@
                             </VisualStateGroup>
                         </VisualStateManager.VisualStateGroups>
 
+                        <Grid
+                            HorizontalAlignment="Left"
+                            VerticalAlignment="Center">
```

Comment:
Is this Grid needed ? can you set the alignment properties on the Rectangle instead ?

Reply:
you're right. Removed.